app/views/DateGoodsUnloadedRejectionView.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.{ErrorSummary, InputDate, Button, HeadingCaption}
+@import java.time.LocalDate
+@import views.html.components.DateErrorSummary
+
+@this(
+        main_template: MainTemplate,
+        errorSummary: DateErrorSummary,
+        formHelper: FormWithCSRF,
+        inputDate: InputDate,
+        button: Button,
+        caption: HeadingCaption
+)
+
+@(mrn: String, arrivalId: ArrivalId, form: Form[LocalDate])(implicit request: Request[_], messages: Messages)
```

Comment:
Is the MRN needed in this view? The arrivalId doesn't appear to be used

Reply:
It's used in  the @caption(mrn) view line 41
The arrivalId is used in the onSubmit(arrivalId) line 37

Reply:
The previous view doesn't seem to pass the mrn into the view so I was presuming the arrivalId would be the caption, though I may be mistaken. I meant the arrivalId wasn't being used in the caption, I wasn't very clear!

Reply:
Mrn appears to have been missed off the njk version